### DISCUSSION TITLE
An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by d1d4r** [vercel/community#4686] [Help] [answered]
### Summary

For Second time !?! vercel verification  show this error message without any explanation, i send email for registration@vercel.com but no one response?!? Please fix this 

### Steps to Reproduce

![IMG_2127](https://github.com/vercel/community/assets/68716408/662deb2a-073b-4a82-8b38-1439ecfd665c)

**Comment by swarnava** [comment, marked as answer]
Our team will reach out via appropriate channel from registration@vercel.com. Unfortunately, we cannot check this via this medium.
